karma.conf.js

```diff
@@ -18,6 +18,7 @@ module.exports = function(config) {
       'misc/test-lib/jquery-1.8.2.min.js',
       'node_modules/angular/angular.js',
       'node_modules/angular-mocks/angular-mocks.js',
+      'misc/test-lib/angular-sanitize.js',
```

Comment:
Should change this to use `node_modules/angular-sanitize/angular-sanitize.js` and modify `package.json` to include angular-sanitize as a module since we changed this in https://github.com/angular-ui/bootstrap/commit/5536e9660e6653ad5e780ce99692789d7bed6fcc so that we don't need to include the angular files in this repository.